### PR TITLE
Update CHANGES for v4.0.0

### PR DESCRIPTION
**Pull request by joe-singleton-itdev**
I've been looking at all the commits between the v3.0.0 tag and current master on [this page](https://github.com/ITDevLtd/MCVirt/compare/v3.0.0...master), and have added everything I think is needed.

A lot of the work done was updating things to work with the Pyro stuff so I don't think needs to be included in the release notes.

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# 4.0.0
+
+* Added 'node' and 'available nodes' to the output of  the 'info' command.
+* Added the ability to remotely start/stop/reset VMs.
+* Added functionality to allow the selection of the driver used for hard drives.
+* Update network module to start network on creation and destroy on deletion.
+* Added ability to switch ISOs whilst VM is running.
+* Implemented an MCVirt daemon that accepts commands through RPC.
```
Comment (joe-singleton-itdev) [inline CHANGES]: Is there a better way of explaining this?

**Comment by itdevreviewboard** [comment]
Can one of the MCVirt admins verify this patch?

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+* Implemented SSL support instead of using SSH.
+* Implemented a user authentication system that is synchronised across nodes.
+* Added the ability to create and remove users.
+* Implemented logging.
+* Add check to ensure that the cluster IP address has been set and that the local hostname resolves to this.
+* Add check of MCVirt version of nodes when running a command.
```
Comment (MatthewJohn) [inline CHANGES]: some of these were part of 3.x releases

File: CHANGES [hunk 3]
```diff
@@ -1,3 +1,34 @@
+# 4.0.0
+
+* Implemented an MCVirt daemon that accepts commands through RPC.
+* Implemented SSL support instead of using SSH.
```
Comment (MatthewJohn) [inline CHANGES]: This is not strictly true. SSH was for communications more than security... although i did that as well..
I would say switch to using 'RPC-over-SSL' from 'a custom command script using SSH'

File: CHANGES [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+* Implemented a user authentication system that is synchronised across nodes.
+* Added the ability to create and remove users.
+* Implemented logging.
+* Add check to ensure that the cluster IP address has been set and that the local hostname resolves to this.
```
Comment (MatthewJohn) [inline CHANGES]: You seem to switch from using 'add' to 'added' and such quite a bit :)

**Pull request merged**